sonar-xml-plugin/src/test/resources/checks/HardcodedCredentialsCheck/web-application/Machine.config

```diff
@@ -0,0 +1,14 @@
+<configuration>
+  <system.web>
+    <authentication mode="Forms">
+      <forms name="customer_login" timeout="30" loginUrl="~/WebGoatCoins/CustomerLogin.aspx" requireSSL="false" protection="All" path="/">
+        <credentials passwordFormat="Clear">
+          <user name="admin" password="admin" /> <!-- Noncompliant {{"password" detected here, make sure this is not a hard-coded credential.}} -->
+                         <!--^^^^^^^^^^^^^^^^ -->
+          <user name="mario" password="luigi" /> <!-- Noncompliant -->
```

Comment:
we should add some tests with:
password="${var}"
password="prefix_#{var}"
password="{{var }}_suffix"